kol/onshape/converter.py

```diff
@@ -952,6 +1038,27 @@ def save_urdf(self) -> None:
                 self.override_torques,
             )
 
+        def read_config_from_json(file_path):
+            with open(file_path, "r") as file:
+                data = json.load(file)
+
+            update_dict = data.get("update_dict", {})
+            override = data.get("override", [])
+            joint_limits = data.get("joint_limits", {})
+            new_torques = data.get("new_torques", {})
+
+            return update_dict, override, joint_limits, new_torques
+
+        if self.config_path is not None:
+            update_dict, override, joint_limits, new_torques = read_config_from_json(self.config_path)
+            update_joints(
+                self.output_dir / f"{robot_name}.urdf",
+                name_dict=update_dict,
+                override_fixed=override,
+                override_limits=joint_limits,
+                override_torques=new_torques,
+            )
+
```

Comment:
oemgaconf for typed configs is fine - https://omegaconf.readthedocs.io/en/2.3_branch/